langchain/vectorstores/pinecone.ts

```diff
@@ -25,7 +25,7 @@ export class PineconeStore extends VectorStore {
     this.textKey = textKey;
   }
 
-  async addDocuments(documents: Document[], ids?: string[]): Promise<void> {
+  async addDocuments(documents: Document[], ids?: string[]): Promise<any> {
```

Comment:
The correct type is `Promise<import("axios").AxiosResponse<UpsertResponse, any>>` but I don't want to import `axios`

Reply:
Just wondering, whether we should wait for newer version of the official repo, which drops the `axios` dependency entirely, which blocks other non-Node environments.

https://github.com/pinecone-io/pinecone-ts-client/issues/5

Reply:
yeah that's a great point. I'm running this with Vercel Next.js API route which I believe is Node so the code works for me.

The other option might be support both until the official one drops `axios`? I'm open for suggestions.

Reply:
One option (that I plan to use for openai adapter too) is to do this https://github.com/openai/openai-node/issues/30#issuecomment-1342350056

Reply:
See #118 for an implementation of this for the openai adapter

Reply:
Not an issue anymore because we are using fetch now.